.github/workflows/test-integration.yaml

```
@@ -4,9 +4,15 @@ on:
   pull_request:
     branches:
       - '*'
+  paths-ignore:
+    - 'docs/**'
+    - '**.md'
   push:
     branches:
       - 'gh-actions'
+  paths-ignore:
+    - 'docs/**'
+    - '**.md'
```

This would allow prepare release PRs to be ignored:
```suggestion
    - '**.md'
    - 'script/gcg/**'
```